.github/workflows/vagrant-up.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: vagrant-up
```

Comment:
I would name this something else, as well as the name of `vagrant-up.yml` - lets use something that better describes the purpose of this test